### DISCUSSION TITLE
Timepicker without AM/PM

### DISCUSSION THREAD

**Discussion by alexandremjacques** [t1m0n/air-datepicker#622] [Q&A]
Is there a way to set the timepicker to 24h? It seems that there is a missing doc section: https://air-datepicker.com/docs?scrollTo=locale

Then click: https://air-datepicker.com/docs#locale-detail (`#locale-detail`) doesn't exists.

Thanks for such good datepicker.

**Comment by t1m0n** [comment]
@alexandremjacques Hello! Thank you for pointing out the lack of documentation. I will fix this. For use 24h mode just set desired timeFormat with capital `H` letter  - https://air-datepicker.com/ru/docs?scrollTo=timeFormat. For example `timeFormat: 'HH:mm'` will set format to 24h mode.
